src/analytics/event.ts

```diff
@@ -694,4 +701,12 @@ export type EventProperties = {
     eventSentAfterMs: number;
     available_data: { description: boolean; image_url: boolean; floorPrice: boolean };
   };
+
+  [event.tokenList]: {
```

Comment:
wondering if the properties associated with this event should be distinct user properties instead. wdyt @DanielSinclair

Reply:
This approach is fine, I think. We generally just want to understand how often users are seeing assets missing metadata on various screens. We can group those back to uniques to understand number of users if we need to.